pkg/alerts/channels.go

```diff
@@ -38,8 +40,9 @@ func (alerts *Alerts) GetChannel(id int) (*Channel, error) {
 			return channel, nil
 		}
 	}
-
-	return nil, fmt.Errorf("no channel found for id %d", id)
+	return nil, &errors.ErrorNotFound{
```

Comment:
I know we've already implemented `errors.ErrorNotFound`, but it feels weird to read vs `errors.NotFound` / `errors.UnexpectedStatusCode`.

Before this PR, I only see a few references to them:

```
internal/http/client.go:		return nil, &errors.ErrorNotFound{}
internal/http/client_test.go:	assert.IsType(t, &errors.ErrorNotFound{}, err)

internal/http/client.go:		return nil, &errors.ErrorUnexpectedStatusCode{
internal/http/client_test.go:	assert.Equal(t, err.(*errors.ErrorUnexpectedStatusCode).Err, "error message")
internal/http/client_test.go:	assert.Equal(t, err.(*errors.ErrorUnexpectedStatusCode).Err, "error message")
internal/http/client_test.go:	assert.IsType(t, &errors.ErrorUnexpectedStatusCode{}, err)
```

Reply:
I updated the names of these types to omit the "Error" prefix.